<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WINDOWS-SBM5E6C" failures="0" tests="2" name="org.Selenium.batchNo32.baseClass.BaseClass" time="0.173" errors="1" timestamp="2022-07-18T20:51:22 IST" skipped="1">
  <testcase classname="org.Selenium.batchNo32.baseClass.BaseClass" name="method1" time="0.000">
    <skipped/>
  </testcase> <!-- method1 -->
  <system-out/>
  <testcase classname="org.Selenium.batchNo32.baseClass.BaseClass" name="intiliaseFiles" time="0.173">
    <error message="class org.Selenium.batchNo32.Utility.Utility (in unnamed module @0x76a4ebf2) cannot access class jdk.internal.org.jline.utils.Log (in module jdk.internal.le) because module jdk.internal.le does not export jdk.internal.org.jline.utils to unnamed module @0x76a4ebf2" type="java.lang.IllegalAccessError">
      <![CDATA[java.lang.IllegalAccessError: class org.Selenium.batchNo32.Utility.Utility (in unnamed module @0x76a4ebf2) cannot access class jdk.internal.org.jline.utils.Log (in module jdk.internal.le) because module jdk.internal.le does not export jdk.internal.org.jline.utils to unnamed module @0x76a4ebf2
at org.Selenium.batchNo32.Utility.Utility.getDatetime(Utility.java:159)
at org.Selenium.batchNo32.Constants.Constant.<clinit>(Constant.java:17)
at org.Selenium.batchNo32.baseClass.BaseClass.intiliaseFiles(BaseClass.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- intiliaseFiles -->
  <system-out/>
</testsuite> <!-- org.Selenium.batchNo32.baseClass.BaseClass -->
